common/lib/xmodule/xmodule/edxnotes_utils.py

```diff
@@ -10,7 +10,7 @@ def edxnotes(cls):
     """
     Conditional decorator that loads edxnotes only when they exist.
     """
-    if "edxnotes" in sys.modules:
+    if "lms.djangoapps.edxnotes" in sys.modules:
         from edxnotes.decorators import edxnotes as notes
```

Comment:
should probably use `from lms.djangoapps.edxnotes.decorators import edxnotes as notes`